src/server/exotics/origin/is-designated-premises/index.js

```diff
@@ -1,8 +1,46 @@
-import { Page } from '~/src/server/common/model/page/page-model.js'
+import { QuestionPage } from '~/src/server/common/model/page/question-page-model.js'
+import { ExoticsQuestionPageController } from '~/src/server/exotics/question-page-controller.js'
+import { RadioButtonAnswer } from '~/src/server/common/model/answer/radio-button/radio-button.js'
+import { animalsOnPremisesPage } from '../animals-on-premises/index.js'
 
-// STUB PAGE
-export class IsDesignatedPremisesPage extends Page {
+/** @import { RadioButtonConfig } from '~/src/server/common/model/answer/radio-button/radio-button.js' */
+/** @import { ServerRegisterPluginObject } from '@hapi/hapi' */
+
+const questionKey = 'isDesignatedPremises'
+
+export class Answer extends RadioButtonAnswer {
+  /** @type { RadioButtonConfig } */
+  static config = {
+    payloadKey: questionKey,
+    hint: 'A designated premises has government approval to undertake certain animal-related activities, such as moving eggs.',
+    options: {
+      yes: { label: 'Yes' },
+      no: { label: 'No' },
+      unknown: { label: "I don't know" }
+    },
+    validation: {
+      empty: 'Enter what animals are on the premises'
```

Comment:
I think this is a ticket bug - sorry about this.  

It should be 'Select if the premises is designated'